README.md

```diff
@@ -164,20 +164,34 @@ private:
 }; 
 ```
 ### Rule 13 
-**Rule:** Functions names will be composed entirely of alphanumeric symbols. Each word, except the first, will start with a capital letter (sometimes known as “camel case.”).  
+**Rule:** Non-member function names will be composed entirely of alphanumeric symbols. Each word, except the first, will start with a capital letter (sometimes known as “camel case.”).  
 
 **Rationale:** Style, easy differentiation of functions, type definitions, and variables. 
 
 **Example 1** 
 ```C++
+int exampleFunction(int x);
+```
+
```

Comment:
Not a bad idea. I have seen this done with a leading _ instead of capital letter. Could look ugly though.